packages/components/organisms/f-checkout/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-checkout",
   "description": "Fozzie Checkout – Fozzie Checkout Component",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
Do we need to bump the minor?  As it's just refactoring would bumping the build number be enough?

Reply:
Can do!